.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+    "tabWidth": 4,
```

Comment:
Thanks for adding this file! One minor ask, let's set `tabWidth` to `2`

Reply:
Added it.